src/synthesiser/Synthesiser.cpp

```diff
@@ -216,16 +216,16 @@ void Synthesiser::emitCode(std::ostream& out, const RamStatement& stmt) {
 
                 switch (rel.getAttributeTypes()[column][0]) {
                     case 'f':
-                        supremum = "ramBitCast<RamFloat>(MIN_RAM_FLOAT)";
-                        infimum = "ramBitCast<RamFloat>(MAX_RAM_FLOAT)";
+                        supremum = "ramBitCast<RamDomain>(MIN_RAM_FLOAT)";
```

Comment:
Note that if the output type can be deduced, the `RamDomain` template parameter can be left out: `ramBitCast(MIN_RAM_FLOAT)`. In this case, though, I think the type helps readability.